packages/snaps-cli/src/cmds/manifest/manifestHandler.test.ts

```diff
@@ -73,7 +73,7 @@ describe('manifest', () => {
     await (manifestModule as any).handler({ ...(foobar as any) });
     expect(manifestHandlerMock).toHaveBeenCalledWith(foobar);
     expect(console.error).toHaveBeenCalledTimes(1);
-    expect(process.exit).toHaveBeenCalledWith(1);
+    expect(process.exitCode).toStrictEqual(1);
```

Comment:
Since we removed the expect statement for the process.exit mock, can we just remove that spy on line 71?